catalog/ui/catalog-ui-search/src/main/webapp/js/model/Workspace.collection.js

```diff
@@ -81,7 +81,8 @@ module.exports = Backbone.Collection.extend({
         }
         var queryForWorkspace = new Query.Model({
             title: title,
-            cql: cqlQuery
+            cql: cqlQuery,
```

Comment:
Similarly, should we be using the user's defaults for searches here?

Reply:
Hmm...maybe we should be using user defaults here. Will that end up impacting the underlying list queries negatively?